### PR TITLE
Symfony profiler page

### PR DESCRIPTION
**Pull request by gam6itko**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| Doc updated   | no
| BC breaks?    | no 
| Deprecations? | no 
| Tests pass?   | yes/no
| Fixed tickets | #812
| License       | MIT

Adds `data_collector` to Symfony profiler.

Depends on https://github.com/schmittjoh/serializer/pull/1316

Features:
- collects profile data for every jms_serializer run.
- collects profile data for all triggered and not triggered event listeners
- collects profile data for all triggered and not triggered handlers
- collects profile data for all loaded metadata via Annotation, Xml, Yaml drivers

I would appreciate any feedback

image

image

image

image

image

### CODE REVIEW

**Comment by goetas** [comment]
The logo you have used is the one for http-plug, can you please use the jms logo? https://github.com/schmittjoh/serializer/blob/master/doc/logo.png

**Comment by goetas** [comment]
@gam6itko FYI https://github.com/schmittjoh/serializer/releases/tag/3.13.0 has been released

**Comment by gam6itko** [comment]
should we up required  "jms/serializer" to "^3.13"?

**Comment by goetas** [comment]
> should we up required "jms/serializer" to "^3.13"?

yes

File: Debug/EventDispatcher/TraceableEventDispatcher.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            foreach ($listeners as $listener) {
+                [$object, $method] = $listener[0];
+                if (is_string($object) && $this->container->has($object)) {
+                    $object = $this->container->get($object);
+                }
+                if ($object instanceof TraceableEventListener) {
+                    yield $event => [
+                        'listener'  => $object->getInnerClass(),
+                        'method'    => $method,
+                        'format'    => $listener[2],
```
Comment (goetas) [inline Debug/EventDispatcher/TraceableEventDispatcher.php:90]: you should  add add `?? null` for `format` and `interface` since they might be not set

**Comment by goetas** [comment]
replaced by https://github.com/schmittjoh/JMSSerializerBundle/pull/858

**Pull request closed without merging**